### PR TITLE
Fix/tbd 9558): correct cxf lib names

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml [hunk 1]
```diff
@@ -309,7 +309,7 @@
...
+            libraries="azure-storage-1.2.0.jar;commons-cli-1.2.jar;commons-codec-1.9.jar;commons-collections-3.2.1.jar;commons-configuration-1.6.jar;commons-lang-2.6.jar;commons-logging-1.2.jar;cxf-rt-bindings-xml-3.3.4.jar;cxf-core-3.3.4.jar;cxf-rt-frontend-jaxrs-3.3.4.jar;cxf-rt-transports-http-3.3.4.jar;guava-12.0.1.jar;hadoop-auth-2.7.1.2.4.1.0-327.jar;hadoop-common-2.7.1.2.4.1.0-327.jar;hadoop-hdfs-2.7.1.2.4.1.0-327.jar;hadoop-mapreduce-client-common-2.7.1.2.4.1.0-327.jar;hadoop-mapreduce-client-core-2.7.1.2.4.1.0-327.jar;hadoop-mapreduce-client-jobclient-2.7.1.2.4.1.0-327.jar;hadoop-yarn-api-2.7.1.2.4.1.0-327.jar;hadoop-yarn-client-2.7.1.2.4.1.0-327.jar;hadoop-yarn-common-2.7.1.2.4.1.0-327.jar;hadoop-yarn-server-web-proxy-2.7.1.2.4.1.0-327.jar;htrace-core-3.2.0-incubating.jar;jackson-core-asl-1.9.14-TALEND.jar;jackson-mapper-asl-1.9.14-TALEND.jar;javax.ws.rs-api-2.0-m10.jar;json_simple-1.1.jar;org.apache.commons.httpclient_3.1.0.v201012070820.jar;parquet-hadoop-bundle-1.6.0.jar;protobuf-java-2.5.0.jar;slf4j-api-1.7.5.jar;slf4j-log4j12-1.7.5.jar;snappy-java-1.0.5.jar;talend-bigdata-launcher-1.1.0-2016040.jar;talend-bigdata-launcher-1.1.0-20160405.jar;wsdl4j-1.6.3.jar">
```
Comment (amedvedenko) [inline main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml]: I don't see cxf-core-3.3.4.jar is declared somewhere in this file. Where does it come from?

**Comment by sponomarova** [comment]
> I don't see changes for Qubole and Databricks, could you please confirm that changes aren't need to be done for these distributions? Reporter mentioned these distributions in the list.

Initially databricks and Qubole didn't have any cxf-rt-core && cxf-api dependencies.
Since the goal of this work item is to upgrade cxf-xxx version - not do add new dependencies that's why you don't see any changes in these distributions regarding cxf-rt-core && cxf-api.

File: main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml (hunk 1 above)
Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml]: I mean that I don't see it's declared here so IMO it's useless to be declared here and can be removed. 
But maybe it's considered as declared in compoent(s)? I don't think so but if it's just let me know)

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml]: Removed

**Pull request merged**